api/tests/core/users/test_api.py

```diff
@@ -620,6 +625,73 @@ def test_with_set_postal_code(self):
         assert updated_user.departementCode == "75"
 
 
+class DepositWithIncidentsTest:
+    def test_deposit_bookings_with_incident(self):
+        deposit = users_factories.DepositGrantFactory()
+
+        bookings_factories.BookingFactory(deposit=deposit, amount=20)
+        bookings_factories.UsedBookingFactory(deposit=deposit, amount=45)
+
+        booking_reimbursed_2 = bookings_factories.ReimbursedBookingFactory(deposit=deposit, amount=40)
+        booking_reimbursed = bookings_factories.ReimbursedBookingFactory(deposit=deposit, amount=30)
+        incident = finance_factories.FinanceIncidentFactory()
+
+        finance_factories.IndividualBookingFinanceIncidentFactory(booking=booking_reimbursed, incident=incident)
+
+        # partial booking finance incident
+        finance_factories.IndividualBookingFinanceIncidentFactory(
+            booking=booking_reimbursed_2, incident=incident, newTotalAmount=3000
+        )
+
+        finance_api.validate_finance_incident(incident, False)
```

Comment:
[détail / style] On peut nommer les arguments quand des constantes (comme `True` ou `False`) leur sont passées, ça aide le lecteur à comprendre quand il ne connaît pas par coeur la signature de la fonction. :)

```suggestion
        finance_api.validate_finance_incident(incident, force_debit_note=False)
```

idem dans les autres tests